completion_aggregator/signals.py

```diff
@@ -106,25 +101,10 @@ def completion_updated_handler(signal, sender, instance, created, raw, using, up
         instance.course_key,
         instance.block_key,
     )
-
-    if not Aggregator.objects.filter(
-            user=instance.user,
-            course_key=instance.course_key,
-            aggregation_name='course',
-            last_modified__gte=instance.modified).exists():
-
-        try:
-            update_aggregators.delay(
-                username=instance.user.username,
-                course_key=six.text_type(instance.course_key),
-                block_keys=[six.text_type(instance.block_key)],
-            )
-        except ImportError:
-            log.warning("Completion Aggregator is not hooked up to edx-plaform.")
-
-
-def get_active_users(course_key):
-    """
-    Return a list of users that have Aggregators in the course.
-    """
-    return get_user_model().objects.filter(aggregator__course_key=course_key).distinct()
+    models.StaleCompletion.objects.create(
+        username=instance.user.username,
+        course_key=instance.course_key,
+        block_key=instance.block_key
+    )
+    if not settings.COMPLETION_AGGREGATOR_ASYNC_AGGREGATION:
```

Comment:
On my solutions (Ginkgo) devstack, where there's no handy django plugin architecture yet, I had to manually set the `settings.COMPLETION_AGGREGATOR_ASYNC_AGGREGATION` variable to avoid getting an error here.

So we'll need to do this on the solutions version bump PR too, or use `getattr(settings, 'COMPLETION_AGGREGATOR_ASYNC_AGGREGATION', something)` here instead.

Reply:
Oh, good point.